src/components/OasisAccount.jsx

```diff
@@ -1,17 +1,27 @@
-import React, { PureComponent } from 'react';
-import { PropTypes } from 'prop-types';
+import React, {PureComponent} from 'react';
+import {PropTypes} from 'prop-types';
 // import ImmutablePropTypes from 'react-immutable-proptypes';
 
 import styles from './OasisAccount.scss';
+import CSSModules from 'react-css-modules';
 
 const propTypes = PropTypes && {};
 const defaultProps = {};
 
 class OasisAccount extends PureComponent {
   render() {
     return (
-      <div className={styles.base}>
-        OasisAccount
+      <div styleName="OasisAccount">
+        <span> Account: </span>
+        <div styleName="AccountSelection">
+          <select>
+            {/*TODO: Get those accounts from the store of web3 directly or passed as a prop*/}
```

Comment:
We should never get data directly from DOM. So we will just have it via the props.